### PR TITLE
allow changing of progress color

### PR DESCRIPTION
**Pull request by tonytamsf**
This allows me to change the progress color from blue -> yellow -> red -> violet almost like a countdown and warning

see screenshot
screen shot 2014-05-10 at 9 16 59 pm

### CODE REVIEW

File: MRCircularProgressView/MRCircularProgressView/MRCircularProgressView.m [hunk 1]
```diff
@@ -126,7 +126,8 @@ - (CAShapeLayer *)shapeLayer
         _shapeLayer.speed=1.0f;
         [self.layer addSublayer:_shapeLayer];
     }
-    
+    _shapeLayer.strokeColor = self.progressColor.CGColor;
```
Comment (martinezdelariva) [inline MRCircularProgressView/MRCircularProgressView/MRCircularProgressView.m:130]: The color is already assigned at line 122.

**Comment by martinezdelariva** [comment]
You can check in the sample code, a progress with red color and with tint color inherit from main view.

**Comment by tonytamsf** [comment]
my use case is that I want to change the color at certain percentage of completion of the duration.

the current code only honors one color and doesn't let me change after progress starts.

sorry I was not clear

**Comment by martinezdelariva** [comment]
Now, it makes sense!.
Then, do you mind delete the color assignment at line 122 in order to don't duplicate code? 
So, I'll merge the pull request.
Thanks!

**Comment by tonytamsf** [comment]
```
Then, do you mind delete the color assignment at line 122 in order to don't duplicate code? 
```

Removed duplicate line and added comment

**Pull request merged**